conda_build/post.py

```diff
@@ -144,10 +144,10 @@ def compile_missing_pyc():
                            '-q', '-x', 'port_v3', sp_dir])
 
 
-def post_process(files, preserve_egg_dir=False):
+def post_process(files, preserve_egg_dir=False, pyc_compilation=True):
     remove_easy_install_pth(files, preserve_egg_dir=preserve_egg_dir)
     rm_py_along_so()
-    if config.CONDA_PY < 30:
+    if config.CONDA_PY < 30 and pyc_compilation:
```

Comment:
Why is compilation only done for Python 2?  @ivoflipse you're perhaps not the right one to ask, since you're only maintaining this behavior, but maybe @ilanschnell or @groutr know?